library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraphTest.java

```diff
@@ -69,6 +67,67 @@ public class GafferPopGraphTest {
         }
     };
 
+    private static final Configuration TEST_CONFIGURATION_2 = new BaseConfiguration() {
+        {
+            this.setProperty(GafferPopGraph.OP_OPTIONS, new String[] {"key1:value1", "key2:value2" });
+            this.setProperty(GafferPopGraph.USER_ID, USER_ID);
+            this.setProperty(GafferPopGraph.DATA_AUTHS, new String[]{AUTH_1, AUTH_2});
+            this.setProperty(GafferPopGraph.GRAPH_ID, "Graph1");
+            this.setProperty(GafferPopGraph.STORE_PROPERTIES, GafferPopGraphTest.class.getClassLoader().getResource("gaffer/store.properties").getPath().toString());
+        }
+    };
+
+    private static final Configuration TEST_CONFIGURATION_3 = new BaseConfiguration() {
+        {
+            this.setProperty(GafferPopGraph.OP_OPTIONS, new String[] {"key1:value1", "key2:value2" });
+            this.setProperty(GafferPopGraph.USER_ID, USER_ID);
+            this.setProperty(GafferPopGraph.DATA_AUTHS, new String[]{AUTH_1, AUTH_2});
+        }
+    };
+
+
+    @Test
+    public void shouldConstructGafferPopGraphWithOnlyConfig() {
+        // Given
+        final User expectedUser = new User.Builder()
+                .userId(USER_ID)
+                .dataAuths(AUTH_1, AUTH_2)
+                .build();
+
+        // When
+        final GafferPopGraph graph = GafferPopGraph.open(TEST_CONFIGURATION_2);
+
+        // Then
+        final Map<String, Object> variables = graph.variables().asMap();
+        assertThat(variables.get(GafferPopGraphVariables.USER)).isEqualTo(expectedUser);
+
+        final Map<String, String> opOptions = (Map<String, String>) variables.get(GafferPopGraphVariables.OP_OPTIONS);
+        assertThat(opOptions.get("key1")).isEqualTo("value1");
+        assertThat(opOptions.get("key2")).isEqualTo("value2");
+        assertThat(opOptions.size()).isEqualTo(2);
+        assertThat(variables.size()).isEqualTo(3);
+    }
+
+    @Test
+    public void shouldConstructGafferPopGraphWithConfigFile() {
+        // Given
+        final User expectedUser = new User.Builder()
+                .userId(USER_ID)
+                .build();
+
+        // when
+        final GafferPopGraph graph = GafferPopGraph.open(GafferPopGraphTest.class.getClassLoader().getResource("gafferpop-tinkerpop-modern.properties").getPath().toString());
```

Comment:
It's already a String.
```suggestion
        final GafferPopGraph graph = GafferPopGraph.open(GafferPopGraphTest.class.getClassLoader().getResource("gafferpop-tinkerpop-modern.properties").getPath());
```